docs/requirements.txt

```diff
@@ -1,3 +1,6 @@
 sphinx<7.2  # See #2023
 sphinx_rtd_theme
 sphinx_last_updated_by_git
+
+# Optional: link Doxygen docs from sphinx
+git+https://github.com/lmoureaux/doxylink
```

Comment:
I'm not sure we should rely on something external to the Longturn set of repos

Reply:
Ok. How about an update to the comment to the same so we know we need to change to upstream at some point in the future?

Reply:
We can also fork it within the org.

Reply:
Meanwhile my [PR upstream](https://github.com/sphinx-contrib/doxylink/pull/63) has been reviewed.